src/app/store/collectibles/selectors.ts

```diff
@@ -0,0 +1,5 @@
+import { useSelector } from '../index';
+import type { CollectibleDetails } from './state';
+
+export const useOneCollectibleDetailsSelector = (slug: string): CollectibleDetails | undefined =>
```

Comment:
better rename to `useCollectibleDetailsSelector` or `useCurrentCollectibleDetailsSelector `